frontend/public/components/notification-drawer.tsx

```diff
@@ -66,10 +66,10 @@ const AlertEmptyState: React.FC<AlertEmptyProps> = ({ drawerToggle }) => (
   </EmptyState>
 );
 
-const actionAlerts: ActionAlertType[] = [
+export const actionAlerts: ActionAlertType[] = [
```

Comment:
Not new, but this should really be a map keyed by alert name.

`alertActions` would be a better name.

Reply:
fixed